src/Box/index.js

```diff
@@ -1,52 +1,65 @@
 import classNames from "classnames";
 import PropTypes from "prop-types";
 
-const Box = ({
+const Box = React.forwardRef(({
   flex,
   alignItems,
   justifyContent,
-  backgroundColor,
```

Comment:
@maxweldsouza removing this prop forces a white background on Box that can't be overridden. 
Example of dashboard's callback page: 
![Dashboard - Maxihost 2021-09-24 at 12 26 17 AM](https://user-images.githubusercontent.com/3950005/134613533-901d9856-0bde-4dc3-a8f7-f2351b45b08e.jpg)

Reply:
I've put in a fix here. https://github.com/Maxihost/metal-ui/pull/264